app/models/flood_risk_engine/organisation.rb

```diff
@@ -2,6 +2,7 @@ module FloodRiskEngine
   class Organisation < ActiveRecord::Base
     belongs_to :contact
     has_one :enrollment, dependent: :restrict_with_exception
+    has_many :partners
```

Comment:
I can't see an alternative that adds any value without adding complication and/or abstraction. I'm open to suggestions.